extensions/amp-graphiq/0.1/validator-amp-graphiq.protoascii

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-graphiq
```

Comment:
The format for the script include changed recently to simplify things. Please replace this section with:

```
tags: {  # amp-graphiq
  html_format: AMP
  tag_name: "SCRIPT"
  satisfies: "amp-graphiq extension .js script"
  requires: "amp-graphiq"
  extension_spec: {
    name: "amp-graphiq"
    allowed_versions: "0.1"
    allowed_versions: "latest"
  }
  attr_lists: "common-extension-attrs"
}
```